### PR TITLE
NMS-15512: Add the missing entries to pom.xml file

### PR DESCRIPTION
**Pull request by mershad-manesh**
Adding a couple of properties to the main `pom.xml` file

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,8 +4,41 @@
...
+    </licenses>
+
+    <scm>
+        <url>https://github.com/OpenNMS/opennms-velocloud-plugin</url>
+        <connection>scm:git:git@github.com:OpenNMS/opennms-velocloud-plugin.git</connection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <developers>
+        <developer>
```
Comment (christianpape) [inline pom.xml:25]: Please include all contributors here, see https://github.com/OpenNMS/opennms-velocloud-plugin/graphs/contributors

**Review by christianpape — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (fooker) [reply pom.xml:25]: IMO, we should not list anything here at all. The `developers` tag is meant to list responsible persons which are about to contact about the project. We have other mechanisms to handle such situations. I see them contrary to our project management methods that we have applied.

Comment (mershad-manesh) [reply pom.xml:25]: We don't necessary need to list all the developers. We need to have at least 1 developer listed (who might get contacted about the project,) to satisfy the requirements should we decide to publish this plugin to Maven Central.

Comment (christianpape) [reply pom.xml:25]: Ok. Got it.

**Review by christianpape — APPROVED** [review]

**Pull request merged**